react/src/screens/Login/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+
+import {validateEmail, validatePasswordLength, validatePasswordContent} from '../../utils/validations';
+import InputWideWithHeader from '../../components/InputWideWithHeader';
+
+import './styles.css';
+import strings from './strings';
+
+class Login extends Component {
+  state = {email: '', password: '', errorEmail: '', errorPassword: ''};
+  submitHandler = e => {
+
+    let errorEmail, errorPassword = "";
+    errorEmail = validateEmail(this.state.email);
+    errorPassword = validatePasswordLength(this.state.password) || validatePasswordContent(this.state.password);
+
+    if(!errorEmail && !errorPassword){
+      sessionStorage.setItem('user_session', this.state.email);
+    }else{
+      this.setState({errorEmail, errorPassword});
```

Comment:
Just checking, if none of these validations fails, these will be populated with `false`. Since `falsy` values are ignored by `React`, we may:
* Convert `false` to `''` or
* Declare `errorEmail` and `errorPassword` as `const`
The first one is 'safer', the second one allows us to write less code by using a `reduce` function